themeServer.js

```diff
@@ -21,17 +21,15 @@ try {
 try {
   // Get Local Host Address
   Object.keys(ifaces).forEach(function (ifname) {
-    let found = false
-
-    ifaces[ifname].forEach(function (iface) {
+    for (const iface of ifaces[ifname]) {
       if (iface.family !== 'IPv4' || iface.internal !== false) {
         // skip over internal (i.e. 127.0.0.1) and non-ipv4 addresses
-        return
+        continue
+      } else {
+        address = iface.address
+        break
       }
```

Comment:
I think this can be refactored a little further into a single `if` statement. Try
`if (iface.family === 'IPv4' || iface.internal === false) {`
`address = iface.address`
`break`
`}`
With this, the for loop on line 24 will continue without the need to write `continue` explicitly.

Reply:
Good catch.  Changes made accordingly.